src/docs/patterns/navigation.mdx

```diff
@@ -3,11 +3,26 @@ import PermanentImg from '../../app/assets/design-patterns/navigation/navigation
 import TemporaryImg from '../../app/assets/design-patterns/navigation/navigation-temporary.png';
 import PersistentImg from '../../app/assets/design-patterns/navigation/navigation-persistent.png';
 
+<!-- New Images -->
+
+import ProductExample1 from '../../app/assets/design-patterns/navigation/Product Example 1.png';
+import ContentCover1 from '../../app/assets/design-patterns/navigation/Content Cover 1.png';
+import ContentCover2 from '../../app/assets/design-patterns/navigation/Content Cover 2.png';
+import GenericExample1 from '../../app/assets/design-patterns/navigation/Generic Example 1.png';
+import NestedItemExample from '../../app/assets/design-patterns/navigation/Nested Item Example.png';
+import NestedItemStandards from '../../app/assets/design-patterns/navigation/Nested Item Standards.png';
+import ExpandableAccountInfo from '../../app/assets/design-patterns/navigation/Expandable Account Information.png';
+import DividerImage from '../../app/assets/design-patterns/navigation/Divider Image.png';
+
 <!-- KEYWORDS: design pattern navigation drawer menu -->
 
 # Navigation Patterns
 
-Navigation is the way in which users access the different areas of your application. This is typically accomplished through a side drawer navigation panel with a list of links to the various pages in your application.
+<ImageGrid images={[ProductExample1]} regularWidth />
+
+Navigation is the way in which users access the different areas of your application. Main navigation is typically accomplished through a side drawer navigation panel with a list of links to the various pages in your application. PX Blue applications can have individual page or feature navigation as well. This can be accomplished by the use of tabs that can run across the top of your application.
+
+Keep in mind that these navigation patterns are tailored for PX Blue application use. PX Blue will be introducing traditional website navigation patterns in the near future.
```

Comment:
Keep the intro short and sweet to introduce what navigation is. The detailed explanations of drawer/tab can go in sections below.

"Navigation is the way in which users are able to access the different parts of your application."